rhobs/olm/manifests/operator/kustomization.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- example/rbac/prometheus-operator/prometheus-operator-deployment.yaml
+- example/rbac/prometheus-operator/prometheus-operator-cluster-role-binding.yaml
+- example/rbac/prometheus-operator/prometheus-operator-cluster-role.yaml
+- example/rbac/prometheus-operator/prometheus-operator-service-account.yaml
+
+namePrefix: rhobs-
+
+patches:
+  - patch: |-
+      apiVersion: apps/v1
+      kind: Deployment
+      metadata:
+        name: prometheus-operator
+      spec:
+        selector:
+          matchLabels:
+            app.kubernetes.io/part-of: rhobs-prometheus-operator
+        template:
+          metadata:
+            annotations:
+              target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
+            labels:
+              app.kubernetes.io/part-of: rhobs-prometheus-operator
+          spec:
+            containers:
+              - name: prometheus-operator
+                image: quay.io/rhobs/obo-prometheus-operator:v0.64.0-rhobs1
```

Comment:
`0.64.1-rhobs3`

Reply:
I have added changing this as part of `make_olm_bundle`